src/lib/bootloader/stage1_proposal.rb

```diff
@@ -95,6 +95,14 @@ def propose
 
     private
 
+      def any_boot_flag_partition?(disk_name)
+        disks = devicegraph.disks.with(name: disk_name)
+        legacy_boot = disks.first.partition_table.partition_legacy_boot_flag_supported?
```

Comment:
well, if we use for booting any disk without partition table, then we have serious problem. But I can change it to false if  there is no boot_flag. But still if disk won't have partition table, then bootloader later blown up